crates/but-core/tests/core/diff/worktree_changes.rs

```diff
@@ -798,7 +798,7 @@ fn symlink_to_file_in_index() -> Result<()> {
                     old_lines: 1,
                     new_start: 1,
                     new_lines: 1,
-                    diff: "@@ -1,1 +1,1 @@\n-target\n+content\n\n",
+                    diff: "@@ -1,1 +1,1 @@-target+content\n",
```

Comment:
I'm not sure that this looks correct.

Reply:
Thanks for helping with this! This should now be addressed by the follow up at #6129.